contrib/network/connection_balance/dlb/test/config_test.cc

```diff
@@ -50,11 +53,16 @@ TEST_F(DlbConnectionBalanceFactoryTest, MakeCustomConfig) {
   envoy::extensions::network::connection_balance::dlb::v3alpha::Dlb dlb;
   dlb.set_id(10);
   dlb.set_max_retries(12);
+  dlb.set_fallback_policy(envoy::extensions::network::connection_balance::dlb::v3alpha::
+                              Dlb_FallbackPolicy_ExactConnectionBalance);
 
   makeDlbConnectionBalanceConfig(typed_config, dlb);
   verifyDlbConnectionBalanceConfigAndUnpack(typed_config, dlb);
   EXPECT_EQ(10, dlb.id());
   EXPECT_EQ(12, dlb.max_retries());
+  EXPECT_EQ(envoy::extensions::network::connection_balance::dlb::v3alpha::
+                Dlb_FallbackPolicy_ExactConnectionBalance,
```

Comment:
Still mangling the end (here and elsewhere)

Reply:
Updated.